### PR TITLE
Adds several new emote sounds

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

Adds several new emotes from citadel, Sunset, and Coyote Bayou

## Why It's Good For The Game

Fluff and flavor! Tested and confirmed functional

## Changelog
```changelog
add: several emote sounds
```

### CODE REVIEW

**Review by fludd12 — APPROVED** [review]
Approved, assuming the "getkers" part is intentional!

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -615,15 +619,65 @@
...
+			message = "lets out a howl!"
+			m_type = 2
+			var/soundpick = pick('zzzz_modular_occulus/sound/voice/coyoteawoo.ogg','zzzz_modular_occulus/sound/voice/coyoteawoo2.ogg', 'zzzz_modular_occulus/sound/voice/coyoteawoo3.ogg','zzzz_modular_occulus/sound/voice/coyoteawoo4.ogg','zzzz_modular_occulus/sound/voice/coyoteawoo5.ogg')
+			playsound(src, soundpick, 50, 1, -1)
+		if("foxbark")
+			message = "lets out a foxy bark!"
+			m_type = 2
+			playsound(src, 'zzzz_modular_occulus/sound/voice/foxbark.ogg', 50, 1, -1)
+		if("gecker")
+			message = "getckers!"
```
[the remark this answers is not in the document]
Comment (Jamini) [reply code/modules/mob/living/carbon/human/emote.dm:640]: Ack, no it's not! I'll fix that

**Review by fludd12 — APPROVED** [review]
Salute the fallen T in geckers.....................

**Comment by Jamini** [comment]
Jamini Getckers.

**Pull request merged**